packages/testring-browser-proxy/src/browser-proxy-controller.ts

```diff
@@ -71,15 +71,23 @@ export class BrowserProxyController extends PluggableModule implements IBrowserP
         this.pendingCommandsPool.clear();
     }
 
-    private onExit = (): void => {
+    private onExit = (code, error): void => {
+        console.log(code, error);
```

Comment:
Remove console.log
Are you commiting without hooks?

Reply:
No, last commits were with hooks